src/libtorch.cc

```diff
@@ -607,6 +614,14 @@ ModelInstanceState::ModelInstanceState(
     if (model_state->ModelConfig().Find("input", &inputs)) {
       expected_input_cnt = inputs.ArraySize();
     }
+
+    triton::common::TritonJson::Value config_batch_inputs;
+    if (model_state->ModelConfig().Find("batch_input", &config_batch_inputs)) {
+      batch_input_count_ = config_batch_inputs.ArraySize();
+      expected_input_cnt += batch_input_count_;
+    } else {
+      batch_input_count_ = 0;
```

Comment:
You can initalize to 0 in the constructor member init list

Reply:
I added an initialisation at the declaration now, let me know if that's okay.